qa/tasks/cephfs/test_misc.py

```diff
@@ -198,6 +198,18 @@ def test_dump_inode_hexademical(self):
         info = self.fs.mds_asok(['dump', 'inode', hex(ino)])
         assert info['path'] == "/foo"
 
+    def test_dispatch_queue_throttle_message(self):
+        """
+        That cluster log a warning when the Dispatch Queue Throttle Limit hits
+        """
+        self.config_set('mds', 'ms_dispatch_throttle_log_interval', 5)
+        self.config_set('mds', 'ms_dispatch_throttle_bytes', 10240)
+
+        # Create files & split across 10 directories, 1000 each.
+        with self.assert_cluster_log("Throttler Limit has been hit. Some message processing may be significantly delayed.",
+                                     invert_match=False, watch_channel="cluster"):
+            for i in range(0, 10):
+                self.mount_a.create_n_files("dir{0}/file".format(i), 1000, sync=False)
```

Comment:
This is a good test but it will only get run in CephFS suites which limits its visibility to breaking changes in the msgr. Can you also add a unit test to `src/test/msgr/test_msgr.cc`?